about.html

```diff
@@ -0,0 +1,140 @@
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+
+                    <div class="nav">
+                        <ul class="nav-ul">
+                            <li class="navigation" id="home"><a href="index.html">Home</a></li>
+                            <li class="navigation"><a href="about.html">About</a></li>
+                            <li class="navigation"><a href="#event">Event</a></li>
+                            <li class="navigation"><a href="#guest">Guest</a></li>
+                            <li class="navigation"><a href="#sponsor">Sponsor</a></li>
+
+                        </ul>
+                    </div>
+
+                    <div class="mobile-menu">
+                        <ul class="menu-list">
+                            <li><a class="mobile-link" href="index.html">Home</a></li>
+                            <li><a class="mobile-link" href="about.html">About</a></li>
+                            <li><a class="mobile-link" href="#event">Event</a></li>
+                            <li><a class="mobile-link" href="#guest">Guest</a></li>
+                            <li><a class="mobile-link" href="#sponsor">Sponsor</a></li>
+                        </ul>
+                    </div>
+
+                    <div class="hamburger">
+                        <div class="bar"></div>
+                        <div class="bar"></div>
+                        <div class="bar"></div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+    <section>
+        <div class="headline-container">
+            <p class="headline-p">"Hello! scholar"</p>
+            <h1>Educational Resources Summit 2023</h1>
+            <p class="headline-summary">
+                Educational resources are used in a learning environment to help and assist with people’s development
+                and
+                learning. We are designed to reinforce learning and in some cases allow people to put their knowledge to
+                the
+                test. Educational resources are brilliant for teachers and educators to help them deliver the best
+                quality
+                lessons.
+            </p>
+            <P class="venue">
+                Please contacts us via Email for any further questions about educational resources 2023!
+                <a href="" class="link">educationalresources@edu.ng</a>
+            </P>
+        </div>
+        <div class="theme">
+            <h2>Theme</h2>
+            <p class="theme-p">
+                "Success occurs when opportunity meets preparation".
+            </p>
+            <p class="summit-logo">
+                <i class="fa-solid fa-graduation-cap cap"></i>
+                <a class="logo-link" href="">Educational Resources</a>
+            </p>
+        </div>
+    </section>
+
+    <hr class="line">
+    <section class="past-summit">
```

Comment:
Nice job on the about page, Let's have a middle section in our about page so as to match the structure of the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)😉

the middle section on the design looks like the image below.. let's add a middle section to the about page😉.

<img width="542" alt="middle" src="https://user-images.githubusercontent.com/64643995/225447109-02723f5e-902d-4608-aa1f-084b03a62d00.png">